pkg/odp/segment_api_manager_test.go

```diff
@@ -306,28 +319,31 @@ func (s *SegmentAPIManagerTestSuite) TestCreateRequestQuery() {
 }
 
 // Tests with live ODP server
-func (s *SegmentAPIManagerTestSuite) TestLiveOdpGraphQL() {
-	segmentsToCheck := []string{"segment-1"}
-	s.config = NewConfig(s.liveOdpAPIKey, s.liveOdpAPIHost, segmentsToCheck)
-	segments, err := s.segmentAPIManager.FetchQualifiedSegments(s.config, "fs_user_id", s.liveOdpValidUserID)
-	s.NoError(err)
-	s.Empty(segments, "none of the test segments in the live ODP server")
-}
+// func (s *SegmentAPIManagerTestSuite) TestLiveOdpGraphQL() {
```

Comment:
is this intentional?

Reply:
Yes, have put these here for debug purposes similar to `swift-sdk`